src/v2i-hub/DsrcImmediateForwardPlugin/src/DsrcMessageManagerPlugin.cpp

```diff
@@ -390,7 +393,7 @@ void DsrcMessageManagerPlugin::SendMessageToRadio(IvpMessage *msg)
 			else
 				os << "Priority=7" << "\n" << "TxMode=CONT" << "\n" << "TxChannel=" << _messageConfigMap[configIndex].Channel << "\n";
 			os << "TxInterval=0" << "\n" << "DeliveryStart=\n" << "DeliveryStop=\n";
-			os << "Signature="<< _signature << "\n" << "Encryption=False\n";
+			os << "Signature= "<< (signState == 1 ? "True" : "False") << "\n" << "Encryption=False\n";
```

Comment:
Need to remove this space to be "Signature=" instead of "Signature= "

Reply:
Good catch

Reply:
Resolved!